pkg/gatherers/workloads/workloads_info.go

```diff
@@ -299,6 +320,44 @@ func gatherWorkloadImageInfo(
 	return imagesDoneCh
 }
 
+// imagesFromChannel drains the channel of any image IDs
+func imagesFromChannel(pendingIDs map[string]struct{}, images map[string]workloadImage, id string, imageCh <-chan string) {
+	for k := range pendingIDs {
+		delete(pendingIDs, k)
+	}
```

Comment:
This looks a little bit weird at the first glance. Do we need to pass the map as the parameter when it's completely deleted then?

Reply:
Nice catch. I overlooked that.